.github/workflows/build.yml

```diff
@@ -68,6 +68,11 @@ jobs:
           ./gradlew --build-cache --info  \
           ${{ inputs.is-public-fork && '--no-scan' || '--scan' }} \
           check -x test
+      - name: Annotate Rat errors
+        if: ${{ failure() }}
```

Comment:
We should only run this on java 21 to avoid duplicate reports from the parallel builds